core/src/main/java/com/linecorp/armeria/common/logging/BuiltInProperty.java

```diff
@@ -167,10 +168,28 @@
      */
     REQ_METHOD("req.method", log -> log.context().method().name()),
     /**
-     * {@code "req.name"} - the human-readable name of the request, such as RPC method name or annotated
-     * service method name. This property is often used as a meter tag or distributed trace's span name.
+     * {@code "req.name"} - the human-readable name of the request, such as:
+     * <ul>
+     *   <li>gRPC - A capitalized method name defined in {@code io.grpc.MethodDescriptor}
+     *       (e.g, {@code GetItems})</li>
+     *   <li>Thrift and annotated service - a method name (e.g, {@code getItems})</li>
+     *   <li>{@link HttpService} - an HTTP method name</li>
+     * </ul>
+     * This property is often used as a meter tag or distributed trace's span name.
      */
     REQ_NAME("req.name", log -> log.isAvailable(RequestLogProperty.NAME) ? log.name() : null),
+    /**
+     * {@code "req.serviceName"} - the human-readable name of the request, such as:
```

Comment:
nit: service name?

Reply:
... name of the service that served the request, ...